tests/src/test/scala/whisk/core/containerpool/test/ContainerPoolTests.scala

```diff
@@ -20,6 +20,7 @@ import java.time.Instant
 
 import scala.collection.mutable
 import scala.concurrent.duration._
+import scala.language.postfixOps
```

Comment:
You'll come to hate `postfixOps` very soon because it induces strange compilation behavior. Personally I avoid it alltogether.